From 1b97e654eecb624125b6bf05059f425ebecafd91 Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Mon, 11 May 2020 14:52:22 +0700
Subject: [PATCH 332/337] arm64: dts: renesas: r8a774e1-hihope-rzg2h: Revert
 disabling mode HS400 for r8a774e1

This commit reverts to support mode HS400 for
eMMC on Hihope G2H platform.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
index f1a6270..201e046 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
@@ -118,10 +118,6 @@
 		"dclkin.0", "dclkin.1", "dclkin.3";
 };
 
-&sdhi3 {
-	/delete-property/ mmc-hs400-1_8v;
-};
-
 &pfc {
 	i2c3_pins: i2c3 {
 		groups = "i2c3";
-- 
2.7.4

